doc/releases/changelog-0.19.0.md

```diff
@@ -0,0 +1,1142 @@
+
+* To standardize across all optimizers, `qml.optimize.AdamOptimizer` now also uses `accumulation` (in form of `collections.namedtuple`) to keep track of running quantities. Before it used three variables `fm`, `sm` and `t`. [(#1757)](https://github.com/PennyLaneAI/pennylane/pull/1757)
+
+<h3>Breaking changes</h3>
+
+- The operator attributes `has_unitary_generator`, `is_composable_rotation`,
+  `is_self_inverse`, `is_symmetric_over_all_wires`, and
+  `is_symmetric_over_control_wires` have been removed as attributes from the
+  base class. They have been replaced by the sets that store the names of
+  operations with similar properties in `ops/qubit/attributes.py`.
+  [(#1763)](https://github.com/PennyLaneAI/pennylane/pull/1763)
+
+* The `qml.inv` function has been removed, `qml.adjoint` should be used
+  instead.
+  [(#1778)](https://github.com/PennyLaneAI/pennylane/pull/1778)
+
+* The input signature of an `expand_fn` used in a `batch_transform`
+  now **must** have the same signature as the provided `transform_fn`,
+  and vice versa.
+  [(#1721)](https://github.com/PennyLaneAI/pennylane/pull/1721)
+
+* The expansion rule in the `qml.metric_tensor` transform has been changed.
+  [(#1721)](https://github.com/PennyLaneAI/pennylane/pull/1721)
+
+  If `hybrid=False`, the changed expansion rule might lead to a changed output.
+
+* The `default.qubit.torch` device automatically determines if computations
+  should be run on a CPU or a GPU and doesn't take a `torch_device` argument
+  anymore.
+  [(#1705)](https://github.com/PennyLaneAI/pennylane/pull/1705)
+
+* The utility function `qml.math.requires_grad` now returns `True` when using Autograd
+  if and only if the `requires_grad=True` attribute is set on the NumPy array. Previously,
+  this function would return `True` for *all* NumPy arrays and Python floats, unless
+  `requires_grad=False` was explicitly set.
+  [(#1638)](https://github.com/PennyLaneAI/pennylane/pull/1638)
+
+* The operation `qml.Interferometer` has been renamed `qml.InterferometerUnitary` in order to
+  distinguish it from the template `qml.templates.Interferometer`.
+  [(#1714)](https://github.com/PennyLaneAI/pennylane/pull/1714)
+
+* The `qml.transforms.invisible` decorator has been replaced with `qml.tape.stop_recording`, which
+  may act as a context manager as well as a decorator to ensure that contained logic is
+  non-recordable or non-queueable within a QNode or quantum tape context.
+  [(#1754)](https://github.com/PennyLaneAI/pennylane/pull/1754)
+
+* Templates `SingleExcitationUnitary` and `DoubleExcitationUnitary` have been renamed
+  to `FermionicSingleExcitation` and `FermionicDoubleExcitation`, respectively.
+  [(#1822)](https://github.com/PennyLaneAI/pennylane/pull/1822)
+
+<h3>Deprecations</h3>
+
+* The `default.tensor` device from the beta folder has not been maintained in
+  years and is deprecated. It will be removed in future releases.
+  [(#1851)](https://github.com/PennyLaneAI/pennylane/pull/1851)
```

Comment:
```suggestion
* The `default.tensor` device from the beta folder is no longer maintained
  and has been deprecated. It will be removed in future releases.
  [(#1851)](https://github.com/PennyLaneAI/pennylane/pull/1851)
```